src/index.html

```diff
@@ -51,6 +51,125 @@ <h1 class="banner-title">Rule the world as a Professional Real Estate Consultant
                 </div>
             </div>
         </section>
+
+        <!-- blog -->
+        <section class="container blog">
+            <div class="blog-cpn features">
```

Comment:
rename to the obvious class name. Avoid abbreviating the class name. Change blog-cpn to blog-wrapper